components/Location.js

```diff
@@ -14,6 +15,13 @@ class Location extends React.Component {
     showTitle: true,
   };
 
+  isGoogleMapsAvailable() {
+    if (typeof google === 'object' && typeof google.maps === 'object') {
```

Comment:
Looks like our linter doesn't like it, better check that `window.google`.

Reply:
Also, `google.maps.places.AutocompleteService` availability is what we want to check.